### PR TITLE
Updated homepage content + new Firefox Reality page

### PR DESCRIPTION
**Pull request by slightlyoffbeat**

### CODE REVIEW

**Comment by slightlyoffbeat** [comment]
This PR is not updating, and is probably related to the various refactoring of this repo. Closing and opening a new PR.

File: src/styles/_config.scss [hunk 1]
```diff
@@ -27,7 +27,7 @@ $breakpoints: (
 
 // Type
 // -------------------------
-@import url('https://fonts.googleapis.com/css?family=Zilla+Slab|Zilla+Slab+Highlight');
+@import url('https://fonts.googleapis.com/css?family=Zilla+Slab|Zilla+Slab+Highlight|Open+Sans');
```
Comment (cvan) [inline src/styles/_config.scss]: I assume this has been carefully and intentionally designed, but it's worth mentioning these fonts add up, especially on mobile, slowing down the page load times, FOUT, etc.

File: src/pages/firefox-reality/components/_built.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+// images
+import bg from '../img/built/bg.jpg';
+import apainter from '../img/built/apainter.png';
```
Comment (cvan) [inline src/pages/firefox-reality/components/_built.js]: are these being imported as data URIs?

File: src/pages/firefox-reality/components/_built.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import kid from '../img/built/kid.png';
+
+const blastStyle = {
+  backgroundImage: `url(${bg})`,
+  backgroundSize: 'cover',
```
Comment (cvan) [inline src/pages/firefox-reality/components/_built.js]: perhaps put this as the default in the stylesheet?

File: src/styles/components/_featured.scss [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    margin-bottom: space(4);
+  }
+
+  @include bp(large) {
+  }
+}
+
+.featured__logo {
+  margin-left: auto;
+  margin-right: auto;
```
Comment (cvan) [inline src/styles/components/_featured.scss]: could condense to one line: `margin: 0 auto;`

File: src/styles/components/_featured.scss [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  display: block;
+  height: 250px;
+}
+
+.featured__link {
+  color: #b3b3ff;
+  border-bottom: 1px solid #b3b3ff;
```
Comment (cvan) [inline src/styles/components/_featured.scss]: feel free to define this colour once as a CSS/SASS var

File: src/styles/components/_featured.scss [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+  margin-bottom: space(2);
+  padding: space(5);
+  color: $white;
+  text-align: center;
+  overflow: hidden;
+  display: block;
+  transition: opacity 0.3s ease;
+
+  &:hover {
+    color: white;
```
Comment (cvan) [inline src/styles/components/_featured.scss]: I notice `$white` is used above.

File: src/styles/components/_featured.scss [hunk 7]
```diff
@@ -0,0 +1,58 @@
+.featured {
+  background-image: linear-gradient(to bottom, #381863 0%, #000000 80%);
```
Comment (cvan) [inline src/styles/components/_featured.scss]: `#000` is shorter

File: src/styles/components/_featured.scss [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+  font-weight: bold;
+  width: 320px;
+  font-family: 'Zilla Slab', $sans-serif;
+  background: #f7296d;
+  position: relative;
+  text-align: center;
+  line-height: 50px;
+  letter-spacing: 0.5px;
+  color: $white;
+  box-shadow: 0px 1px 13px rgba(8, 8, 8, 0.97);
```
Comment (cvan) [inline src/styles/components/_featured.scss]: `0px` -> `0`

File: src/styles/components/_footer.scss [hunk 9]
```diff
@@ -3,16 +3,21 @@
...
 .footer__logo {
-  height: space(4);
   width: auto;
+  max-width: 80%;
 }
 
 .footer__title {
   font-weight: 600;
-  letter-spacing: .5px;
+  letter-spacing: 0.5px;
```
Comment (cvan) [inline src/styles/components/_footer.scss]: it was actually better/shorter before 😆

File: src/styles/components/_header.scss [hunk 10]
```diff
@@ -17,31 +17,42 @@
...
   @include bp(medium) {
+    float: right;
     display: block;
   }
 }
 
 .nav__list {
+  display: flex;
+  justify-content: center;
+  margin: 16px 0 0 0;
```
Comment (cvan) [inline src/styles/components/_header.scss]: minor: could change to `margin: 16px 0 0;`

File: src/styles/components/_header.scss [hunk 11]
```diff
@@ -17,31 +17,42 @@
...
+    padding: 0;
+  }
 
   a {
     color: $black;
     font-weight: 600;
     border-bottom: 0 solid rgba(0, 0, 0, 0);
-    transition: border-bottom 200ms $base-ease,
-                opacity 200ms $base-ease;
+    transition: border-bottom 200ms $base-ease, opacity 200ms $base-ease;
```
Comment (cvan) [inline src/styles/components/_header.scss]: if you want to save 2 characters (twice): `200ms` -> `.2s`

File: src/styles/style.scss [hunk 12]
```diff
@@ -47,6 +47,12 @@
 @import 'components/experience';
 @import 'components/additional-resources';
 @import 'components/about';
+@import 'components/featured';
+
+// Components
+// -------------------------
+@import 'pages/firefoxreality';
```
Comment (cvan) [inline src/styles/style.scss]: nice modular structure :+1:

File: src/layouts/img/logo.svg [hunk 13]
```diff
@@ -1 +1,31 @@
...
+                                    </g>
+                                </g>
+                            </g>
+                            <text id="Mixed-Reality" font-family="ZillaSlab-Regular, Zilla Slab" font-size="29" font-weight="normal" fill="#FFFFFF">
```
[the remark this answers is not in the document]
Comment (cvan) [reply src/layouts/img/logo.svg]: I reopened the original issue so it doesn't get forgotten about:

https://github.com/MozillaReality/vr-mozilla-org/issues/43#issuecomment-422384648

**Pull request merged**